src/client/pages/Welcome.tsx

```diff
@@ -13,19 +13,26 @@ import { sendOphanReferrerEvent } from '@/client/lib/ophan';
 type Props = {
   submitUrl: string;
   fieldErrors: FieldError[];
-  viewId?: string;
+  refViewId?: string;
+  refPage?: string;
```

Comment:
We have to call the prop `refPage` rather than `ref` as `ref` is a React specific prop: https://reactjs.org/docs/refs-and-the-dom.html

Reply:
changed to `refUrl` as that makes more semantic sense